rails-commenting.rb

```diff
@@ -48,15 +48,15 @@ def update
   def destroy
     @post = BlogPost.find(params[:id])
     if @post.destroy
-      # ---8)
+      # ---8) HTTP verb Delete. After the Destroy Method has been implimented the user will be redirected to the show page where they can view all the blog posts(minus the post that was deleted)
       redirect_to blog_posts_path
     end
   end
 
-  # ---9)
+  # ---9) By using the key word private, it makes it so the strong params can't be accessed by the rest of the methods, unless it is called specifically in the method itself.
```

Comment:
The `private` keyword is used to make anything below it only accessible in the scope of the current controller